drivers/bridge/bridge_test.go

```diff
@@ -111,18 +111,35 @@ func TestCreateFullOptionsLabels(t *testing.T) {
 		t.Fatalf("Failed to setup driver config: %v", err)
 	}
 
+	bndIPs := "127.0.0.1"
+	nwV6s := "2100:2400:2600:2700:2800::/80"
+	gwV6s := "2100:2400:2600:2700:2800::25/80"
```

Comment:
This seems to be an unassigned network. I suggest you to use: [`2001:db8::/32` as per RFC 3849](https://tools.ietf.org/html/rfc3849)